include/imgproc/dct_hash.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+// Standard C++ includes
+#include <iostream>
+#include <string>
+#include <algorithm>
+#include <bitset>
+// Standard C includes
+#include <cmath>
+
+//opencv
+#include <opencv2/opencv.hpp>
+
+namespace DCTHash
+{
+    //! computes the DCT hash
+    inline static std::bitset<64> computeHash(const cv::Mat &in) {
+
+        cv::Mat dct_mat;
+        cv::dct(in, dct_mat);
+        cv::Mat rect(dct_mat,{ 0, 0, 8, 8});                // we only need the low 8x8 frequencies
+        std::sort(rect.begin<float>(),rect.end<float>());   // sorting to get median value
+        const float median= rect.at<float>(-1+rect.size().height/2,rect.size().width-1); 
```

Comment:
The median is calculated wrongly here. There are 64 values here, so the median will be (element[31] + element[32]) / 2 (with zero-based indexing)